_jobs/executive-assistant.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: job_post
+title: Executive Assistant Business Development
+---
```

Comment:
You can add `open: true` here to make it show up in the "Accepting applications" section

Reply:
```suggestion
open: true
---
```

Reply:
woah this GH interface is cool. but ha, yes just found that in the code! thanks!